C2LC-574: Added credits for third-party images.

the-t-in-rtf opened the pull request:
See [C2LC-574](https://issues.fluidproject.org/browse/C2LC-574) for details.

the-t-in-rtf commented:
I was able to resolve the build failures by slightly modifying our configuration for [the "no-inline-markdown" rule](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md033) to specifically allow the HTML `br` element.

This seems a good balance, as we get consistency with the rest of the docs, and leave ourselves the option to use breaks within GfM tables elsewhere in the docs.

michelled commented:
This looks great overall! I noticed that a lot of the Flickr pages say the images are in the public domain - I think you double checked those and put a CC license because it was appropriate, right? 

A couple links to check:
The link to Petr Kratochvil goes to Andre Deak
Is there no link for Jerome Biederman?

A couple names to fix:
D Coetzee is Chiara Coetzee
romainguy is the username of Romain Guy

the-t-in-rtf commented:
> This looks great overall! I noticed that a lot of the Flickr pages say the images are in the public domain - I think you double checked those and put a CC license because it was appropriate, right?

Their description for both licenses is simply "public domain", but if you hover over the link, [CC0 ones](https://creativecommons.org/publicdomain/zero/1.0/) have "zero" and [Public Domain Mark 1.0 ones](https://creativecommons.org/publicdomain/mark/1.0/) have "mark" instead.  I'll double-check just to be sure.

the-t-in-rtf commented:
I found three or four that had the wrong license and fixed those.

the-t-in-rtf commented:
> A couple links to check: The link to Petr Kratochvil goes to Andre Deak

Bad copypasta from the previous line, fixed.

> Is there no link for Jerome Biederman?

There wasn't, fixed now.
 
> A couple names to fix: D Coetzee is Chiara Coetzee romainguy is the username of Romain Guy

Fixed.

the-t-in-rtf commented:
I reviewed the whole list of names briefly, the only one we can't find a better name for is "lettie".

michelled commented on docs/image-credits.md:
```diff
@@ -0,0 +1,54 @@
+| [![Statue of Liberty](images/statue-of-liberty.jpg)](https://flic.kr/p/21RsmfF) | Statue of Liberty | [Claudia Saffar](https://www.flickr.com/photos/153667655@N08/) | [Public Domain Mark 1.0](https://creativecommons.org/publicdomain/mark/1.0/) |
+| [![Stonehenge](images/stonehenge.jpg)](https://flic.kr/p/2kULoDE) | Stonehenge | [Michael Sutton](https://www.flickr.com/photos/192368704@N03/) | [Public Domain Mark 1.0](https://creativecommons.org/publicdomain/mark/1.0/) |
+| [![Sukhoi Superjet 100](images/sukhoi-superjet-100.jpg)](https://flic.kr/p/ULMDzk) | Sukhoi Superjet 100 | [Bartlomiej Mostek](https://www.flickr.com/photos/barteq24/) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
+| [![Sydney Opera House](images/sydney-opera-house.jpg)](https://flic.kr/p/6oXs7X) | Sydney Opera House | [D Coetzee](https://www.flickr.com/photos/29507259@N02) | [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) |
```
I'm still seeing the old names here - am I looking in the wrong place?

the-t-in-rtf replied:
Good catch, a few changes were not saved before the last push, my IDE was cheerfully displaying the "not saved dot" when I started this morning.  It's fixed now.

michelled replied:
Looks good to me! Thanks!
